doc/releases/pacific.rst

```diff
@@ -133,6 +133,20 @@ RBD block storage
 RGW object storage
 ~~~~~~~~~~~~~~~~~~
 
+* Initial support for S3 Select. See :ref:`s3-select-feature-table` for supported queries.
+
+* Bucket notification topics can be configured as ``persistent``, where events
+  are recorded in rados for reliable delivery.
+
+* Bucket notifications can be delivered to SSL-enabled AMQP endpoints.
+
```

Comment:
we now support AWS compliant API to get topic attributes: GetTopicAttributes

Reply:
@yuvalif your text from PendingReleaseNotes is already under the `Notable Changes` section, so i didn't repeat it under rgw:
```
* An AWS-compliant API: "GetTopicAttributes" was added to replace the existing "GetTopic" API. The new API
  should be used to fetch information about topics used for bucket notifications.
```

Reply:
sorry, i missed that. thanks!